stix_shifter_modules/splunk/stix_translation/json/to_stix_map.json

```diff
@@ -266,10 +266,23 @@
       "references": "process-exec"
     }
   ],
-  "process_hash": {
-    "key": "file.hashes",
-    "object": "process-exec",
-    "transformer": "SplunkHash"
+  "Hashes": {
+    "SHA256": {
+      "key": "file.hashes.SHA256",
+      "object": "file"
+    },
+    "MD5": {
+      "key": "file.hashes.MD5",
+      "object": "file"
+    },
+    "SHA1": {
+      "key": "file.hashes.SHA1",
+      "object": "file"
+    },
```

Comment:
correct the hash property names. here's an example: https://github.com/opencybersecurityalliance/stix-shifter/blob/2250c474cda99783dac83fca3b69d60f362d9433/stix_shifter_modules/qradar/stix_translation/json/to_stix_map.json#L343